latch_cli/extras/common/config/utils.py

```diff
@@ -220,6 +221,13 @@ def get_preamble(typ: Type, *, make_optionals: bool = False) -> str:
             [get_preamble(t, make_optionals=make_optionals) for t in get_args(typ)]
         )
 
+    if issubclass(typ, Enum):
```

Comment:
Would we be able to just import the enum from latch_metadata instead of reconstructing?